shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1211,6 +1217,40 @@ - (void)invalidateTouchRateCorrectionVSyncClient {
   _touchRateCorrectionVSyncClient = nil;
 }
 
+#pragma mark - Stylus Events
+
+#pragma clang diagnostic push
+
+- (void)pencilInteractionDidTap:(UIPencilInteraction*)interaction API_AVAILABLE(ios(13.4)) {
+  flutter::PointerData pointer_data = [self generatePointerDataAtLastMouseLocation];
+
+  switch (UIPencilInteraction.preferredTapAction) {
+    case UIPencilPreferredActionIgnore:
+      pointer_data.preferred_action = flutter::PointerData::PreferredAction::kIgnore;
+      break;
+    case UIPencilPreferredActionShowColorPalette:
+      pointer_data.preferred_action = flutter::PointerData::PreferredAction::kShowColorPalette;
+      break;
+    case UIPencilPreferredActionSwitchEraser:
+      pointer_data.preferred_action = flutter::PointerData::PreferredAction::kSwitchEraser;
+      break;
+    case UIPencilPreferredActionSwitchPrevious:
+      pointer_data.preferred_action = flutter::PointerData::PreferredAction::kSwitchPrevious;
+    default:
+      break;
```

Comment:
if apple come up with a new case, will this cause trouble on the framework side? Or should there be a "kUnknown" type?